examples/first-component/chromatic.config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I just saw this fille was list in `.gitignore`, please check

Reply:
Fixed